src/mps_runner.jl

```diff
@@ -44,10 +44,10 @@ function parse_commandline()
         "--compute-truncation-error"
             help = "whether to compute the error induced by truncation at each step (computationally expensive)"
             action = :store_true # default without this flag is false
-        "--tau"
-            help = "time evolution step size in seconds"
-            arg_type = Float64
-            default = 0.01e-6
+        # "--tau"
```

Comment:
If tau is not used, should it instead be "supported" here but print that it is deprecated?